modules/installation-aws-regions-with-no-ami.adoc

```diff
@@ -1,26 +1,29 @@
 // Module included in the following assemblies:
 //
 // * installing/installing_aws/installing-aws-china.adoc
-// * installing/installing_aws/installing-aws-government-region.adoc
 // * installing/installing_aws/installing-aws-user-infra.adoc
+// * installing/installing_aws/installing-aws-secret.adoc
 
 ifeval::["{context}" == "installing-aws-china-region"]
 :aws-china:
 endif::[]
-ifeval::["{context}" == "installing-aws-government-region"]
-:aws-gov:
+ifeval::["{context}" == "installing-aws-secret-region"]
+:aws-secret:
 endif::[]
+// ifeval::["{context}" == "installing-aws-government-region"]
+// :aws-gov:
+// endif::[]
 
 [id="installation-aws-regions-with-no-ami_{context}"]
-ifndef::aws-china,aws-gov[]
+ifndef::aws-china,aws-secret[]
 = AWS regions without a published {op-system} AMI
-endif::aws-china,aws-gov[]
+endif::aws-china,aws-secret[]
 
-ifdef::aws-china,aws-gov[]
-= Installation requirments
-endif::aws-china,aws-gov[]
+ifdef::aws-china,aws-secret[]
+= Installation requirements
+endif::aws-china,aws-secret[]
 
-ifndef::aws-china,aws-gov[]
+ifndef::aws-china,aws-secret[]
 You can deploy an {product-title} cluster to Amazon Web Services (AWS) regions
 without native support for a {op-system-first} Amazon Machine Image (AMI) or the
 AWS software development kit (SDK). If a
```

Comment:
The following paragraph includes language about copying the AMI from `us-east-1`. That is only applicable to the public partition and not the government partition.

Reply:
I believe we are OK here. For starters, this content is excluded from the gov topic entirely. Where this content is shared, the paragraph is excluded from the china and secret partitions.

Reply:
Ah, OK. I did not consider that this content is not included in the gov topic. Sounds good.